metadata-models/src/main/pegasus/com/linkedin/dataset/FineGrainFieldUpstream.pdl

```diff
@@ -0,0 +1,21 @@
+namespace com.linkedin.dataset
+
+import com.linkedin.common.AuditStamp
+import com.linkedin.common.DatasetUrn
+
+/**
+ * Upstream lineage information about a dataset including the source reporting the lineage
+ */
+record FineGrainFieldUpstream {
+
+  /**
+   * The source field
+   */
+  sourceField: string
```

Comment:
the source field should be an array and include a reference to the dataset the source field belong to. A destination field can be generated from multiple source fields from different datasets (joins for example).

Reply:
hi @hshahoss , I have used https://github.com/linkedin/datahub/issues/1731 to explain how and why I use different models